packages/venia-concept/src/components/ProductImageCarousel/carousel.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .currentImage {
-    background-color: rgb(var(--venia-grey));
+    background-color: white;
```

Comment:
Do we want to create a named CSS variable for this?

Reply:
Sure.

Reply:
(Obviously not for `white`, which is a global constant, but perhaps for how this view uses it.)